release/preview/opensuse423/docker/Dockerfile

```
@@ -0,0 +1,103 @@
+# Docker image file that describes an OpenSuse 42.3 image with PowerShell installed from tar.gz
+# Implemented in two stages, one that downloads and uncompresses the tar.gz
+# The next stage simply copies the files from the first stage,
+# effectively removing the tar.gz and packages that are not needed from the final image.
+
+# Define arg(s) needed for the From statement
+ARG fromTag=42.3
+
+FROM opensuse:${fromTag} AS installer-env
+
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=6.2.0-powershell.3
```

Fix `ARG` value originated from stable release to preview

https://github.com/PowerShell/PowerShell-Docker/blob/554a54d464797db5bfc7134a9479da9ea53c23d9/release/preview/opensuse423/docker/Dockerfile#L13

```suggestion
ARG PS_VERSION=6.2.0-preview.3
```